### PR TITLE
ci: Allow any word in brackets after build to make dependabot work

### PR DESCRIPTION
**Pull request by krille-chan**

### CODE REVIEW

File: .github/workflows/general.yml [hunk 1]
```diff
@@ -13,4 +13,4 @@ jobs:
           fetch-depth: 0
       - name: Check conventional commits
         run: |
-          (git log --pretty=format:%s --no-merges HEAD ^${{github.event.pull_request.base.sha}} |  ( ! grep -v -E '^(ci|fix|feat|chore|test|perf|refactor|style|builds|build\(deps\)|docs): ' ) > tempout ) || ( echo -e '# Not conventional commits:' >> $GITHUB_STEP_SUMMARY; cat tempout >> $GITHUB_STEP_SUMMARY; exit 1 )
+          (git log --pretty=format:%s --no-merges HEAD ^${{github.event.pull_request.base.sha}} |  ( ! grep -v -E '^(ci|fix|feat|chore|test|perf|refactor|style|builds|build\(\w+\)|docs): ' ) > tempout ) || ( echo -e '# Not conventional commits:' >> $GITHUB_STEP_SUMMARY; cat tempout >> $GITHUB_STEP_SUMMARY; exit 1 )
```
[the remark this answers is not in the document]
Comment (krille-chan) [reply .github/workflows/general.yml:16]: Why not just allow any valid `word` there?

[the remark this answers is not in the document]
Comment (krille-chan) [reply .github/workflows/general.yml:16]: I mean sometimes dependabot names it `build(deps)` and in other time it names it `build(deps-dev)`. `build\(_\w+\)?` would not allow those

[the remark this answers is not in the document]
Comment (TheOneWithTheBraid) [reply .github/workflows/general.yml:16]: Were only samples, that's why `\w`. See updated/extended regex proposal.

[the remark this answers is not in the document]
Comment (TheOneWithTheBraid) [reply .github/workflows/general.yml:16]: That one would match both in brackets and the cases of underscore suffixes as you may know them from `build_deps` or something.

[the remark this answers is not in the document]
Comment (krille-chan) [reply .github/workflows/general.yml:16]: I made `builds|build_\w+|build\(\w+\)` for better readability

**Review by techno-disaster — APPROVED** [review]

**Pull request merged**